src/mono/mono/component/mini-wasm-debugger.c

```diff
@@ -1,3 +1,4 @@
+#ifndef HOST_WASI
```

Comment:
Should this be `#ifdef HOST_WASM` instead?

Reply:
I don't think so, when we are compiling wasi HOST_WASM and HOST_WASI are defined.

Reply:
um then should it be `#if defined(HOST_WASM) && !defined(HOST_WASI)`?